docs/tutorials/actions.mdx

```diff
@@ -23,7 +23,7 @@ To facilitate this process, the proposer must specify the stake credential that
 
 **Previous governance action ID**
 
-To prevent unintended conflicts between governance actions of the same type, it is essential that each governance action includes the governance action ID of
+To prevent unintended conflicts between governance actions of the same type, it is essential that some governance action includes the governance action ID of
```

Comment:
```suggestion
To prevent unintended conflicts between governance actions of the same type, some governance actions must include the governance action ID of
```